src/django_idom/utils.py

```diff
@@ -102,11 +113,14 @@ def _get_components(self, templates: set[str]) -> set[str]:
         for template in templates:
             with contextlib.suppress(Exception):
                 with open(template, "r", encoding="utf-8") as template_file:
-                    match = COMPONENT_REGEX.findall(template_file.read())
-                    if not match:
+                    regex_iterable = COMPONENT_REGEX.finditer(template_file.read())
+                    if not regex_iterable:
                         continue
                     components.update(
-                        [group[0].replace('"', "").replace("'", "") for group in match]
+                        [
+                            match.groupdict()["path"].replace('"', "").replace("'", "")
```

Comment:
I'm pretty sure you can just do this if you only need one named group:
```suggestion
                            match.group("path").replace('"', "").replace("'", "")
```